pkg/deploy/tls-secrets.go

```diff
@@ -0,0 +1,247 @@
+//
+// Copyright (c) 2020 Red Hat, Inc.
+// This program and the accompanying materials are made
+// available under the terms of the Eclipse Public License 2.0
+// which is available at https://www.eclipse.org/legal/epl-2.0/
+//
+// SPDX-License-Identifier: EPL-2.0
+//
+// Contributors:
+//   Red Hat, Inc. - initial API and implementation
+//
+package deploy
```

Comment:
I think it is controller logic

Reply:
It is just a package name :)

Reply:
Everything is controller.

Reply:
Why do we need packages then :)

Reply:
In controller package we methods for ReconcileChe structure only.
So, it make sense to move tls-secrets.go to deploy package.
Let's keep as is